src/browser.ts

```diff
@@ -69,7 +72,19 @@ export class GrpcClient {
 
   async createStub(service: protobuf.Service, opts: ClientStubOptions) {
     const authHeader = await this.auth.getRequestHeaders();
-    async function serviceClientImpl(method, requestData, callback) {
+    function serviceClientImpl(method, requestData, callback) {
+      let cancelController, cancelSignal;
+      if (typeof AbortController === 'undefined') {
+        cancelController = undefined;
```

Comment:
Let's make it smaller: they are both `undefined` by default, so we can invert the `if` condition and just assign them if `AbortController` exists. That way, no `else` and the logic will be easier to understand.

Reply:
Done.